lesson_12/structs_java/structs_app/src/main/java/com/codedifferently/lesson12/Stack.java

```diff
@@ -9,18 +9,22 @@ public Stack() {
   }
 
   public void push(int value) {
-    // Your code here
+    ListNode tophat = new ListNode(value);
```

Comment:
Camel case for method variables, here and below.

Reply:
I updated them to be camel case, I forgot bout camel case that's on me.